### ISSUE TITLE
Down arrowhead icons on compose page have no accessible name

### ISSUE THREAD

**Issue by dankeck** [thunderbird/thunderbird-android#8232]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

8.0b1

### Where did you get the app from?

Google Play

### Android version

14

### Device model

Pixel 5

### Steps to reproduce

1. Start the Android TalkBack screen reader.
2. Open Thunderbird Beta.
3. Compose a new email message.
4. Use TalkBack to navigate to the down arrowhead icon next to the From field, or the icon next to the To field.
![Compose screen highlighting a down arrowhead icon next to the From field. The same icon also appears next to the To field.](https://github.com/user-attachments/assets/0066eac9-fbc5-4743-bd1b-317cfb8a2495)

### Expected behavior

TalkBack says the purpose of the button, such as "more" or "expand" or "additional fields".

### Actual behavior

TalkBack says "unlabeled button" and does not convey the purpose of the button.

**Comment by mececeli** [comment]
I would like to work on this issue.

**Comment by Akshaykomar890** [comment]
Hi @kewisch  I'd like to work on this issue. Could you please assign it to me?

**Comment by kewisch** [comment]
Hey folks, we were off towards the end of the year so I couldn't respond earlier. @mececeli have you had success with a PR on this issue? This generally seems simple enough that someone could just go ahead and send a pull request, but of course I don't want to duplicate the work if someone has already started.

**Comment by mececeli** [comment]
> Hey folks, we were off towards the end of the year so I couldn't respond earlier. [@mececeli](https://github.com/mececeli) have you had success with a PR on this issue? This generally seems simple enough that someone could just go ahead and send a pull request, but of course I don't want to duplicate the work if someone has already started.

Hey there! I made a fix on my local, but looks like somebody already raised a PR above, which looks like similar to my fix. Next time I can raise PR for these kind of bugs. See you on the next bugs, happy to contribute!

**Comment by kewisch** [comment]
Thanks for sticking with us, I'm looking forward to see what is next for you!
